products/opensuse/main.pm

```diff
@@ -407,6 +407,15 @@ else {
             set_var('INSTALLONLY', 1);
             loadtest "iscsi/iscsi_client";
         }
+        if (get_var('WIREGUARD_SERVER') || get_var("WIREGUARD_CLIENT")) {
+            set_var('INSTALLONLY', 1);
+            barrier_create('SETUP_DONE',      2) if get_var('WIREGUARD_SERVER');
+            barrier_create('KEY_TRANSFERED',  2) if get_var('WIREGUARD_SERVER');
+            barrier_create('VPN_ESTABLISHED', 2) if get_var('WIREGUARD_SERVER');
```

Comment:
```
if (get_var('WIREGUARD_SERVER')) {
             barrier_create('SETUP_DONE',      2) ;
            barrier_create('KEY_TRANSFERED',  2);
            barrier_create('VPN_ESTABLISHED', 2);
}
```

Reply:
I will just use `IS_MM_SERVER` instead of `HOSTNAME` :)